### PR TITLE
Add artists & secret package for artist & secret single source of truth

### PR DESCRIPTION
**Pull request by maman**
## Summary

Add artist & secrets sharedlib package

### CODE REVIEW

File: common/secrets/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "name": "@koplonesia/secrets",
```
Comment (ans-4175) [inline common/secrets/package.json:2]: cara bikin package.json monorepo gini manual atau gimana sih @maman ?

File: common/secrets/spotify.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const Spotify = require('spotify-web-api-node');
```
Comment (ans-4175) [inline common/secrets/spotify.js:1]: sip nice

**Review by ans-4175 — APPROVED** [review]
LGBT!, eh LGTM!
PR kamu masuk dulu, baru PR-ku diperbaikin ngikutin PR-kamu boss

File: common/secrets/package.json (hunk 1 above)
Comment (maman) [reply common/secrets/package.json:2]: ini manual, via yarn init haha

**Pull request merged**